### PR TITLE
Adds say_mod variable to felinids

### PR DESCRIPTION
**Pull request by Farquaar**
## About The Pull Request
Felinids now "meow" when they speak, in the same way that a lizard hisses or a plasmaman rattles.

Maintainer Approved
dentonapproves

EDIT: Changed the verb to "yaps" instead of "meows". Feel free to drop suggestions, of course.
EDIT2: Apparently you heathen philistines insist that yapping is only done by dogs. I've switched the verb to "blithers". Every day we stray further from God's light.
EDIT3: Changes the verb to "meows" on maintainer request. Like Pontius Pilate, I wash my hands my hands of the matter. **If you have suggestions, get maintainer approval** before bringing it to me.

## Why It's Good For The Game
It can be difficult for asimov silicons to distinguish felinids from normal humans, especially over the radio. This should help rectify that issue.

## Changelog
:cl:
tweak: Felinid speech is now audibly distinct from human speech.
/:cl:

### CODE REVIEW

**Comment by PKPenguin321** [comment]
off topic but this reminds me that we need a speech mod for tongue-less people

**Comment by ExcessiveUseOfCobblestone** [comment]
if someone meows can I paste the copypasta legally everytime they speak?

**Comment by wesoda25** [comment]
this pr is terrible

File: code/modules/mob/living/carbon/human/species_types/felinid.dm [hunk 1]
```diff
@@ -2,6 +2,7 @@
 /datum/species/human/felinid
 	name = "Felinid"
 	id = "felinid"
+	say_mod = "meows"
```
Comment (wesoda25) [inline code/modules/mob/living/carbon/human/species_types/felinid.dm]:
```suggestion
	say_mod = "rasps"
```

**Comment by HomocidlePancakes** [comment]
Fuck. Cats.

**Review by 81Denton — DISMISSED** [review]
Just to clarify - I approved this because it will hopefully reduce the amount of situations where silicons are unsure whether they have a human wearing cat ears (asimov protected, catmins will ban you) or a felinid (no asimov) in front of them.

**Comment by Farquaar** [comment]
People are more than welcome to suggest different verbs. I'm not the biggest fan of felinids, but so long as they are in the game, they should be easily identifiable to Asimov silicons.

**Comment by Tlaltecuhtli** [comment]
merge suggestion then

**Comment by Farquaar** [comment]
> 
> 
> merge suggestion then

If it gets maintainer approval, then sure.

**Comment by necromanceranne** [comment]
Make it 'nyas', you coward.

File: code/modules/mob/living/carbon/human/species_types/felinid.dm (hunk 1 above)
Comment (necromanceranne) [reply code/modules/mob/living/carbon/human/species_types/felinid.dm]: Every felinid is a pack a day smoker.

**Comment by 81Denton** [comment]
@Farquaar dew it

**Comment by 81Denton** [comment]
@Farquaar I'm indifferent about it, if you think it fits then do it

**Comment by InsaneKatrina** [comment]
_Oh crap._

**Comment by necromanceranne** [comment]
I've got it.

'Saysies'

**Comment by Farquaar** [comment]
I'm thinking something along the lines of "yaps" or "chortles"
Rasps is funny, but I can foresee it being used as a good say_mod for people with lung damage in the future.

**Comment by Farquaar** [comment]
> 
> 
> Cats dont yap? 😕

People yap, though.

**Comment by kriskog** [comment]
dogs yap, it doesnt fit.

**Comment by Farquaar** [comment]
> 
> 
> dogs yap, it doesnt fit.

I disagree, but you're more than welcome to bring your own suggestions.

**Comment by necromanceranne** [comment]
Yaps bad awoos begone.

Also yeah uh just a synonym for 'says' might be fine.

**Comment by wesoda25** [comment]
How dare you resolve my suggestion 

Anyways chortles and babbles are better simply because they don’t reward cats with a catlike verb

File: code/modules/mob/living/carbon/human/species_types/felinid.dm (hunk 1 above)
Comment (wesoda25) [reply code/modules/mob/living/carbon/human/species_types/felinid.dm]: That gives me an idea for another pr actually

**Comment by ExcessiveUseOfCobblestone** [comment]
c(h)ats 😏

**Comment by Tlaltecuhtli** [comment]
i used google and it has a bunch of alternatives
waul yaul youl

**Comment by Farquaar** [comment]
> 
> 
> 
> Anyways chortles and babbles are better simply because they don’t reward cats with a catlike verb

Those are essentially synonyms for what's in the PR already. Yapping, babbling, chortling, they're all equally descriptive.

I don't see how felinids yapping is a reward for anyone but silicons. If anything, I'd imagine that some felinid players might be salty that they can't trick borgs into obeying Law 2 orders anymore.

> 
> 
> waul yaul youl

As a native English speaker, I've never heard any one of those words in my life. Maybe "yowl", but that word describes an expression of pain. (i.e. "The man yowled in pain when the car ran over his foot.")

**Comment by cacogen** [comment]
yaps is retarded, just use meows
full disclosure: i do not play catpeople

**Comment by Ty-the-Smonk** [comment]
Cats mew

**Comment by necromanceranne** [comment]
Of all the suggestions, at least 'mews sounds somewhat more tonally appropriate if we have to go with a animal noise for general speech. 

But trying to make it too close to an animal noise is not going to work as a general speech sound. Expresses, conveys, discloses, denotes, suggests (probably an ask verb more) are all much more generic and appropriate for a race that is otherwise human.

**Comment by Farquaar** [comment]
"Mews" is too cutesy. I'm sure most people would rather not see felinids go any further in that direction.

**Comment by wesoda25** [comment]
yeah thats why i said you should just use something passive like chortles/babbles/all the non animalistic ones people suggested

**Comment by Farquaar** [comment]
Can't please everybody I guess.

**Comment by ExcessiveUseOfCobblestone** [comment]
set it to meows or mews and i'll merge it before people start complaining about the door color to the bike shed next.

**Comment by kriskog** [comment]
Dont need to go further than the hops office to hear Ian yap.

**Comment by PKPenguin321** [comment]
It's secret lore, catgirls are secretly made from spliced and modified dog genes

**Comment by trollbreeder** [comment]
Felinids are humans that abuse genetic modifications

**Comment by Iamgoofball** [comment]
I really don't want to weigh in on a catpeople PR because its a huge dramasink for no gain, but:

lizards and plasmamen have completely different vocal cords and mouth shapes than humans do, which would make them sound different

catpeople are literally just humans with a tail and ears

nothing about their physiology would make them sound any different than a human

@ExcessiveUseOfCobblestone idk if you're still planning to merge it, but this should probably be taken into account

like, yeah, a lizard hisses, thats cuz it has a forked tongue, a muzzle/snout, and completely different throat setups

and a skeleton rattles because its a fuckin skeleton, rattle me bones

where do catpeople get the ability to sound different from? ears and tails don't vocalize

**Review by 81Denton — CHANGES_REQUESTED** [review]
Let's just go with "meows", it looks like the most neutral choice and I'm not looking to start The Great Felinid Debate: Part 2135234

**Comment by PKPenguin321** [comment]
They have severe speech impediments due to significant genetic damage. Imagine, if you will a family that has inbred for ten-dozen generations, to the point that their defiled genes will now gladly accept the DNA of a cat, as it no longer sees the difference. The family, too, does not see the difference, as they all either suffer from some notable amount of blindness or are mentally feeble enough to be able to discern a cat from a man. Thanks to space science, we accelerated this process with a time-acceleration chamber, causing centuries of bestiality and inbreeding to occur in a relative time of mere seconds!
This actually led to the creation of the genetics wing. A common misconception is that genetic modifications put man into cat, but in truth, it was to take the cat out of the man.
Anyways, the point is they don't talk quite right good.

**Comment by TheVekter** [comment]
Just fucking keep it as "meows". You dumb cunts will get angry at literally anything involving cats. Zero reason we should be having this discussion.

**Comment by Rohesie** [comment]
Please, do consider whether you want people meowing around.

**Comment by TheVekter** [comment]
> 
> 
> Please, do consider whether you want people meowing around.

You cannot begin to fathom how little I care what verb comes after someone's name when they talk in this game. It helps with visibility for AIs, just do it instead of whinging over the details.

**Comment by Farquaar** [comment]
May God have mercy on all of us.

**Comment by 81Denton** [comment]
Alright gamers

**Pull request merged**